src/lexer.l

```diff
@@ -350,42 +352,42 @@ static int resolve_ir1045(void)
 
 static int parse_id(const char *str)
 {
-   char *p = (yylval.s = xmalloc(strlen(str) + 1));
-   while (*str) {
-      *p++ = toupper((int)*str);
-      str++;
-   }
-   *p = '\0';
+  char *p = (yylval.s = xmalloc(strlen(str) + 1));
+  
+  while (*str) {
+    *p++ = toupper((int)*str);
+    str++;
+  }
+  *p = '\0';
 
-   TOKEN(tID);
+  TOKEN(tID);
```

Comment:
Some of these functions only have whitespace changes - maybe your editor did that automatically? The C code in nvc should be indented with two spaces.